requirements.txt

```diff
@@ -175,8 +175,6 @@ torchvision==0.15.2
 tornado==6.3.3
 tqdm==4.65.0
 traitlets==5.9.0
-transformer-lens @ git+https://github.com/neelnanda-io/TransformerLens.git@218ebd6f491f47f5e2f64e4c4327548b60a093eb
-transformers==4.36.0
```

Comment:
Removing `transformer-lens` makes perfect sense but many of the files above explicitly import the transformers module, shouldn't we keep this in our `requirements.txt`?

Reply:
Also, to test this it would be a good idea to create a blank conda environment and use that to run some simple experiments to discover dependency issues.

Reply:
This code uses a slightly modified version of transformers where I added hooks, which is why i removed transformers from requirements.

Reply:
Do you mean that you have a clone of transformers that you use to make this work? In that case we should include your clone in the `requirements.txt` - you should be able to point `pip` towards a particular commit in your repo the same way that we handled `transformer-lens` above.

Am I understanding that correctly?

Reply:
If I am understanding Jordan correctly. We are actually making a local copy of specific transformer model implementations and modifying those implementations directly right? Therefore we may not need any transformer library specific utilities, but it might be good to think about future use cases where we might.

Reply:
I have added my branch of hugging face transformers to the requirements. I am not entirely sure what you mean @msakarvadia . I have been modifying a branch of hugging face transformers in order to maintain features such as quantization and multi-gpu support, so we still need to import that specific branch of transformers.